packages/talker_flutter/lib/src/ui/talker_monitor/talker_monitor_typed_logs_screen.dart

```diff
@@ -19,7 +19,12 @@ class TalkerMonitorTypedLogsScreen extends StatelessWidget {
     return Scaffold(
       backgroundColor: theme.backgroundColor,
       appBar: AppBar(
-        title: Text('Talker Monitor $typeName'),
+        backgroundColor: theme.backgroundColor,
+        iconTheme: IconThemeData(color: theme.textColor),
+        title: Text(
+          'Talker Monitor $typeName',
+          style: TextStyle(color: theme.textColor),
+        ),
```

Comment:
This changes isn't related to this PR
Can you create another PR with this changes ?